packages/turf-concave/index.d.ts

```diff
@@ -5,8 +5,8 @@ import { Point, Feature, FeatureCollection, MultiPolygon, Polygon, Units} from '
  */
 export default function concave(
     points: FeatureCollection<Point>,
-    maxEdge: number,
     options?: {
-      units?: Units
+        maxEdge: number,
```

Comment:
using the `?` means it's optional, at the moment if someone would enter options, then `maxEdge` would be required.

Fix to:
```ts
options?: {
  maxEdge?: number,
  units?: Units
}